### ISSUE TITLE
Mint 18, Nemo 3.0.6 - moving files around locks up Nemo. 

### ISSUE THREAD

**Issue by trebushet** [linuxmint/nemo#1220]
Updated to Mint 18. File management with Nemo has almost become unusable. It locks up constantly when trying to move files from one folder to another. Out of 1000 pictures files in one folder, tried to move 115 files to another folder by click and dragging. Nemo locked up for close to 5 minutes before it copied half the files, it locked up again for close to another 5 minutes. It finally finished close  to 10 minutes later, when the operation should have taken < 1 second. 

This has been an issue with Nemo for a number of years now, but nobody cares to fix this. It was not happening this bad in Mint 17.x, but it just became a lot worse in Mint 18. Very frustrated with Nemo and its inability to handle large file management.

**Comment by MasterPCUK** [comment]
Got the same issue.

1: Select about 150 items (I had raw photos and about 2Gb in size)
2: Drag selected items to another Nemo window.
3: Result completely locks up Nemo. See screenshot.

I am running fresh Linux Mint 18 install on a decent laptop with SSD and 4GB ram.
![screenshot from 2016-07-21 18-32-41](https://cloud.githubusercontent.com/assets/8301081/17029105/90e70772-4f72-11e6-9dd5-28366df5a94a.png)

**Comment by donlencho** [comment]
Just experienced the same thing, although it's not totally "locked", I can see files being moved one by one in the destination folder, but as described above, very irregularly. Then the destination folder looked empty again (Nemo was just lost I guess) and finally it ended really quickly (It took maybe one minute to move less than ten files, but in the end, the 100+ remaining files were moved in a few seconds). The little file manager window with the progress bar was saying "Preparing to move..." the whole time.
[Of course I'm talking about moving files from one folder to another on the same partition]

**Comment by Co6aka** [comment]
FWIW, this has been going on for at least three years without a fix.

**Comment by feren** [comment]
Not sure, but it may also happen with Nautilus too though...
Also, when it's happened to me, it's locked up too (Not Responding Scenario)

**Comment by Co6aka** [comment]
FWIW, people are reporting the same bug happening with Caja.

**Comment by donlencho** [comment]
Not very "scientific" but I'm also guessing it has something to due with the thumbnails generation. I mostly experienced it with images and videos and I guess it takes time to process. Don't know if that is a possibility for investigation.

**Comment by jkonley** [comment]
I have this same problem on Mint 19.1. Figured it was something in Mint so tried some other fixes online with no luck. Installed Dolphin file manager and problem went completely away.   Was able to copy and paste tons of pics with zero slowdown or lockup.  Annoying that I'm finding that this is an ongoing issue that has not been resolved. Copying files is basic functionality.

**Comment by icarter09** [comment]
Is anyone still experiencing this issue? I was able to drag over 700 pictures (4.2 GB) from one folder to another. Operation took about a minute in total and experienced no lock up by Nemo.

**Comment by trebushet** [comment]
I have not experienced the lockup issue in the last couple years. 

Nemo still crashes when having multiple tabs open and being in "Tree view" while performing copy/move drag and drop jobs between tabs, but that is happening less frequently than in the past, and is probably not related to this issue in this ticket.

**Comment by Minabsapi** [comment]
Found this issue while looking for something else. I can confirm this still happens. Currently sorting a pictures folder of less than 500 files (all less than 10Mo) to dispatch in other folders with CTRL+C/X and drag-and-drops in another nemo window with 4 tabs opened, and after a while nemo hangs/doesn't respond. It eventually became so unbearable I had to stop doing my sorting this way and now I'm moving the files with the terminal
